BitwardenShared/UI/Platform/Application/Extensions/UITabBarControllerExtensionsTests.swift

```diff
@@ -5,7 +5,7 @@ import XCTest
 
 // MARK: - UITabBarControllerTests
 
-class UITabBarControllerTests: BitwardenTestCase {
+class UITabBarControllerExtensionsTests: BitwardenTestCase {
```

Comment:
🔨  ah i missed this, thanks!  small nit, can we update the `MARK` here too?  again, my bad!

Reply:
Ah, good catch!